cfpb/cfgov-refresh
# Extend regdown form fields

Extends regdown pseudo form fields so that they span the full width of the page.

## Changes

- Nests a child `span` SubElement inside the `regdown-form_extend` span.
- Gives regdown paragraphs (and divs) a `regdown-block` class so we can differentiate them from normal paragraphs.
- Updates all the tests to check for the `regdown-block` class.

## Testing

1. Paste [this regdown](https://gist.github.com/contolini/1fe8e711e16285b8ad269dbad1e8de47) into any reg page.
2. View the page and it should match the screenshots below.

## Screenshots

| Before | After |
|--------|-------|
| <img width="676" alt="screen shot 2018-06-05 at 11 13 12 am" src="https://user-images.githubusercontent.com/1060248/40986920-571eda98-68b5-11e8-9d0a-d937464151f6.png">    | <img width="773" alt="screen shot 2018-06-04 at 8 44 41 pm" src="https://user-images.githubusercontent.com/1060248/40986784-06d3a85c-68b5-11e8-823b-9f5de878e76a.png">   |

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
